Sources/Datadog/RUM/RUMVitals/VitalMemoryReader.swift

```diff
@@ -0,0 +1,33 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ */
+
+import Foundation
+
+internal class VitalMemoryReader: VitalReader {
+    static let TASK_VM_INFO_COUNT = MemoryLayout<task_vm_info>.size / MemoryLayout<natural_t>.size
```

Comment:
We don't use uppercase for defining constants - they are reserved for compiler directives. Also, seems that this one can be private:
```swift
private static let task_vm_info_count = MemoryLayout<task_vm_info>.size / MemoryLayout<natural_t>.size
```